filament/backend/src/opengl/ShaderCompilerService.cpp

```diff
@@ -597,6 +612,51 @@ std::string_view ShaderCompilerService::process_GOOGLE_cpp_style_line_directive(
     return { source, len };
 }
 
+// Look up the `source` to replace the number of eyes for multiview with the given
+// number. This is necessary for OpenGL because OpenGL relies on the number
+// specified in shader files to determine the number of views for multiview.
+// This funciton assumes the number of views is a single digit.
+std::string_view ShaderCompilerService::process_OVR_multiview2(OpenGLContext& context,
+        char* source, size_t len, int32_t eyeCount) noexcept {
+    if (context.ext.OVR_multiview2) {
```

Comment:
it looks like this will run pretty much all the time. Is there a way we could skip this for the common case, which is when stereo is not used?

Reply:
It's possible If we store a metadata indicating STE or multiview in Program class. Variant is in filabridge, so it seems not preferable to store it in Program class (filament/backend) as we discussed yesterday. However, we could use a boolean or more generic variable instead.